From: Ben Hutchings <ben@decadent.org.uk>
Date: Sun, 25 Sep 2016 01:42:34 +0100
Subject: uio: Fix ABI change in 4.7.5
Forwarded: not-needed

iov_iter_fault_in_readable() and
iov_iter_fault_in_multipages_readable() are now equivalent, with the
latter name defined as a macro.  Restore it as a real function too for
ABI compatibility.

---
--- a/lib/iov_iter.c
+++ b/lib/iov_iter.c
@@ -301,6 +301,14 @@ done:
 	return wanted - bytes;
 }
 
+/* bwh: Retained for ABI compatibility */
+#undef iov_iter_fault_in_multipages_readable
+int iov_iter_fault_in_multipages_readable(struct iov_iter *i, size_t bytes)
+{
+	return iov_iter_fault_in_readable(i, bytes);
+}
+EXPORT_SYMBOL(iov_iter_fault_in_multipages_readable);
+
 /*
  * Fault in one or more iovecs of the given iov_iter, to a maximum length of
  * bytes.  For each iovec, fault in each page that constitutes the iovec.
